DTS-STN/DECD-Design-System
# Ds 112/colour style

**Description & Background Context:**
Storybook Documentation for Colors based on [Figma design ](https://www.figma.com/file/YFz6bzcX1DSAQfu4BSfm6G/Design-System-Extended?node-id=284%3A41)

**Where should the reviewer start?**
File name: Color.stories.js Color.js

**How should this be tested?**
The reviewer should look at the storybook link in this PR to view the Colors Page and verify the presented information

**Are there any breaking changes?**
No
